ui/dialog.js

```diff
@@ -12,6 +12,8 @@
 //>>description: Displays customizable dialog windows.
 //>>docs: http://api.jqueryui.com/dialog/
 //>>demos: http://jqueryui.com/dialog/
+//>>css.structure: ../themes/base/dialog.css
+//>>css.theme: ../themes/base/theme.css
```

Comment:
Yes, they are. In this case, yeap `button.css` will be included due to the AMD dependency to `button.js`.